### DISCUSSION TITLE
PK-Sim and RNA therapeutics

### DISCUSSION THREAD

**Discussion by cristinavaghi** [Open-Systems-Pharmacology/Forum#1274] [Questions & Problems]
Dear all,

I was wondering whether PK-Sim allows PK modelling of RNA-based therapeutics, and more specifically of antisense oligonucleotides. Are there published examples of such modelling in PK-Sim?

Thank you very much for your help!

**Comment by wilbertdew** [comment]
Hi Christina,

PK-Sim has a large molecule model which has been used for large molecules [with a large range of molecular weights (5 -150 kDa)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5845054/pdf/10928_2017_Article_9559.pdf). The shape and charge of oligonucleotides might require some model adaptations based on observed data, but this would be a good starting point I think. I am not aware of any specific applications of PK-Sim to oligonucleotides but others might be aware? Feel free to reach out for further discussions!
Best,
Wilbert de Witte

**Comment by prvmalik** [comment]
First thing I would try is setting oligonucleotide hydrodynamic radius in the two pore model, adding GFR with fraction 1.0, keeping KdFcRn as very high and then optimizing either specific clearance in the endosome or rate constant for endosomal uptake. This should capture the plasma PK but may not capture the organ specific disposition, most notably the liver. Let me know how it goes.

**Comment by Christoph27** [comment]
For antisense oligonucleotides it might be interesting to consider the combination of selecting the “Is small molecule” property at the “Basic physico-chemisty” tab for the compound and using the model for large molecules and proteins. With that, you allow distribution into cells (as with the small molecule model) but you still have the extravasation barrier from the large molecule model. The endosomal uptake & endosomal clearance is switched off in this case and in addition to the renal clearance (glomerular filtration mentioned by Paul) you would most likely need to add an additional plasma clearance. (The general endosomal clearance is probably anyhow a very minor clearance pathway compared to others for these compounds.). Lipophilicity gets then a parameter which is used in principle in the model and for which you probably won’t have a value. Due to the large size of your drug it might have not a large sensitivity however, just try ...

**Comment by harishkaushikbugworks** [comment]
Hello All,

was wondering whether anyone tried in recent past setting up a PBPK model for ASO therapeutics based on the suggestions mentioned above

It will be of help if it is shared here as well

Regards
Harish

**Comment by SCZ1174** [comment]
Hi, 
I thought these information would be useful.
[10.1007/s10928-021-09761-0](https://link.springer.com/article/10.1007/s10928-021-09761-0) (Modeled with ADAPT5)
[10.1080/17425255.2021.1992382](https://www.tandfonline.com/doi/full/10.1080/17425255.2021.1992382) 

with best,
CZ

**Comment by harishkaushikbugworks** [comment]
Thanks for sharing these articles, i have read these

I was looking for PK-Sim based PBPK model for ASOs and siRNA

It would be helpful if something is developed using OSP framework

Regards
Harish

**Comment by harishkaushikbugworks** [comment]
Hello All,

I reecently came across a publication from Eric Sjorgen  group, attached the paper here
[Whole-Body Physiologically-Based Pharmacokinetic for siRNAs.pdf](https://github.com/user-attachments/files/17990038/Whole-Body.Physiologically-Based.Pharmacokinetic.for.siRNAs.pdf)

They attempted to build some WB-PBPK Model for GA1NC molecules using mobi (suppl file has been provided)

However there is no clarity on whether the model is build first in PK-SIM and then the model was exported into mobi, or directly the model was build in mobi

tried to reach Eric, looks like his email is not working

anyone has any insights it will help 

Regards
Harish

**Comment by wilbertdew** [comment]
almost all models are first built in PK-Sim, otherwise you don't have the
whole-body model structure

Op di 3 dec. 2024 09:59 schreef harishkaushikbugworks <
***@***.***>:

**Comment by Erik-Sjogren** [comment]
Hi Harish,
Apologies for the late response. My primary email address had become obsolete.
The final model is now published and an example model file is available via the sup. material.
https://www.mdpi.com/1999-4923/17/1/69
It is indeed built on the WB-PPBK PK-Sim backbone.
I hope you find it useful.
BR
Erik

**Reply by harishkaushikbugworks** [reply]
Thanks Erik,

It was worth the wait for us, thanks for sharing the link to publication and mobi file, if i may request could you share PK-Sim prioject file, it will be very useful

Regards
Harish

**Reply by Erik-Sjogren** [reply]
Hi Harish,
As the extensions go beyond default PK-Sim implementation there is unfortunately no such file.
Kind regards
Erik

**Comment by Neha1981** [comment]
I was able to predict the plasma PK and when exported to Mobi and added Brain CSF compartment , iam getting error during simulation can any one help here
[Nusinersen PBPK mobi to be shared.zip](https://github.com/user-attachments/files/18534140/Nusinersen.PBPK.mobi.to.be.shared.zip)

Regards
Neha

**Reply by wilbertdew** [reply]
Hi @Neha1981, I had a look at your file, but I am afraid there is no quick fix for the error as far as I can see. I compared your spatial structures with and without CSF, but most parameter values are different, so it seems your spatial structure is built from another spatial structure than the one in your file. I also noticed there are no passive transports defined for the neighbourhoods you included and there are no updated Molecule Start values and parameter start values in your file. All of this contributes to the errors in your file, and the simulation cannot be build. The first thing to pay attention to if you build a simulation is that you switch of the "is present" checkbox for all molecules except for the dummy in the new compartments. However, there is an issue with the parameter values as well, which I believe is due to the difference between the spatial structure building blocks with and without csf. If you want to work with the current project file, I would go back and clone the dummy spatial structure and re-introduce the CSF compartments

**Reply by Neha1981** [reply]
Thanks Wilbert, very helpful suggestions
however, may i request you to help me fix this, as iam new and i could try and develop the model till this stage, mobi is new to me

It will be great help if you could do help me in fixing this and able to predict CSF concentrations

Regards
Neha

**Reply by wilbertdew** [reply]
Hi @Neha1981, we can definitely help you with that! Can you send an email to wilbert.dewitte@esqlabs.com? Then we can set up a meeting and see what is needed
